site/homepage/content/release-notes.md

```diff
@@ -4,6 +4,14 @@ title = "What is new & noteworthy in Hono?"
 description = "Information about changes in recent Hono releases. Includes new features, fixes, enhancements and API changes."
 +++
 
+## 2.3.0
+
+### Fixes & Enhancements
+
+* The MQTT adapter did forward command messages for devices behind gateway when the gateway subscription was targeting
+the gateway itself with topic name `command//[<gateway-id>]/req/#`. This has been fixed and the commands for devices,
+for which the gateway acts on its behalf of, are not forwarded to the gateway.
```

Comment:
The last sentence sounds a bit complicated. I would suggest:
"This has been fixed. For a subscription with such a topic name, only commands for the gateway itself are forwarded, not commands for devices, that the gateway acts on behalf of.

Reply:
Right, your suggestion is better.